example/ndpiReader.c

```diff
@@ -884,6 +890,11 @@ static void parseOptions(int argc, char **argv) {
 
     case 'n':
       num_threads = atoi(optarg);
+      if(num_threads > 1 && enable_malloc_bins == 1)
```

Comment:
Is this check really needed? You are already checking for `if(num_threads > 1 && enable_malloc_bins == 1)` after the switch-case.

Reply:
You are right!